### ISSUE TITLE
Mnist batch size one

### ISSUE THREAD

**Issue by ilmucio** [pytorch/examples#276]
When I try the mnsit example with batch size one .. I got random like accuracy. Is that an expected behavior?

**Comment by ilmucio** [comment]
The default learning rate for batch size one should be lowered to 0.001 to make the network learn... That was suggest to me in the pytorch forum by ptrblck .
